src/components/Coin.js

```diff
@@ -0,0 +1,73 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import { getCoins } from '../redux/coins/coinsSlice';
+
+export default function Coin() {
```

Comment:
- [ OPTIONAL ] Make a try to ES6 arrow function and you will love code readability, you could do `export default () => {}` and import it with the name of your choice or ` const Coin = () => {}` and then `export default Coin` at the end of your file would do the same, compared to function declaration, the arrow function has a Concise syntax and much more ...

Reply:
Thanks.